backend/kernelCI_app/management/commands/helpers/kcidbng_ingester.py

```diff
@@ -47,7 +47,7 @@
 try:
     INGEST_FLUSH_TIMEOUT_SEC = float(os.environ.get("INGEST_FLUSH_TIMEOUT_SEC", "2.0"))
 except (ValueError, TypeError):
-    logger.warning("Invalid INGEST_FLUSH_TIMEOUT_SEC, using default 2.0")
+    logger.warning("Invalid INGEST_FLUSH_TIMEOUT_SEC, using default 5.0")
     INGEST_FLUSH_TIMEOUT_SEC = 5.0
```

Comment:
It seems the value should be `2.0`, if we are going with 5 then line 48 also needs to be updated

Reply:
True, changed all of them to 2.0